Presentation.Web/Models/API/V2/Response/Generic/Validity/ValidityResponseDTO.cs

```diff
@@ -5,10 +5,14 @@ namespace Presentation.Web.Models.API.V2.Response.Generic.Validity
     public class ValidityResponseDTO
     {
         /// <summary>
-        /// Determines if the entity is considered valid. This is computed from both "EnforcedValid" as well as ValidFrom/To
+        /// Determines if the entity is considered valid. This is computed from both "EnforcedValid" as well as ValidFrom/To.
```

Comment:
ValidAccordingToValidityPeriod replaces the ValidFrom/To, since you explain the meaning og ValidAccordingToValidityPeriod in that comment